app/app.js

```diff
@@ -25,7 +27,7 @@ import configureStore from './store';
 
 // Observe loading of Open Sans (to remove open sans, remove the <link> tag in
 // the index.html file and this observer)
-import styles from 'containers/App/styles.css';
+import styles from './containers/App/styles.css';
```

Comment:
I'm don't think that it's because of webpack. The reason it's working would be `app.js` placed in the some folder as `containers`. That's why the `./` could be omitted. However, Eslint will warn that the path can't be resolved. [On line 26th](https://github.com/lvarayut/react-boilerplate/blob/3b19ef54dc5d2db42ae2449aa4bc536b233abc39/app/app.js#L26), we could do `import configureStore from 'store';` as well, we just need to be consistent.